web-gui/src/app/tool-components/participants/participants.component.html

```diff
@@ -4,8 +4,14 @@
   <mat-form-field class="search" appearance="outline">
    <mat-label>Teilnehmer suchen und hinzufügen</mat-label>
     <input matInput (keyup)="applyFilter($event.target.value)" [matAutocomplete]="auto">
+    <input type="text"
+           placeholder="Pick one"
+           aria-label="Number"
+           matInput
+           [formControl]="search"
+           [matAutocomplete]="auto">
     <mat-autocomplete #auto="matAutocomplete" (optionSelected)='addParticipant($event.option.value)' [displayWith]="displayFn">
-      <mat-option *ngFor="let user of allUser" [value]="user">
+      <mat-option *ngFor="let user of searchedUser" [value]="user">
```

Comment:
Can't find the line where you change the state of searchedUser. Is it supposed to be always an empty array?

Reply:
Yes i forgot, that i was working on that, should be fixed now